apps/win32/nuget/Microsoft.FluentUI.FluentTesterWin32.nuspec

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.FluentUI.FluentTesterWin32</id>
+    <version>1.0.0</version>
+    <title>FluentUI-React-Native Win32 Bundle</title>
+    <authors>Microsoft Office CXE</authors>
+    <projectUrl>https://github.com/microsoft/fluentui-react-native.git</projectUrl>
+    <description>This package contains the React Native JS bundle of FluentUI-React-Native's Win32 Test App.</description>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+  </metadata>
+  <files>
+    <file src="..\dist\index.win32.bundle" target="win32" />
```

Comment:
Are the sourcemaps inline?  Or should you publish those too?

Reply:
Also, is this not .jsbundle ?

Reply:
Only ios/macOS use `.jsbundle`.  I'm guessing because `.bundle` means something else on macOS.  All the other platforms use `.bundle` for their bundles.

Reply:
Ah, makes sense! The existence of [NSBundle](https://developer.apple.com/documentation/foundation/nsbundle) explains that.

Reply:
After bundling, I don't see any sourcemap files. I believe it's inline. Is there a way to double-check? How can I be sure? I've already tested out the published NuGet package with ReactTest and everything seems to work